app/views/admins/edit_custom.html.erb

```diff
@@ -0,0 +1,9 @@
+<h1>Change Account Password</h1>
```

Comment:
`edit_password ` can be directly used with out edit custom ?

Reply:
edit_password is used with existing edit action.